src/transform/__tests__/tsTransformModule.test.ts

```diff
@@ -0,0 +1,148 @@
+import { tsTransformModule } from '../tsTransformModule';
+import { initTypescriptConfig } from '../../tsconfig/configParser';
+import * as ts from 'typescript';
+
+describe('tsTransformModule test', () => {
+  let compilerOptions: ts.CompilerOptions;
+
+  const testTransformer = context => {
+    const visit: ts.Visitor = node => {
+      return ts.visitEachChild(node, child => visit(child), context);
+    };
+    return node => ts.visitNode(node, visit);
+  };
+
+  beforeAll(() => {
+    const config = initTypescriptConfig({});
+    compilerOptions = config.compilerOptions;
+  });
+
+  it('should transform module without any custom transformers', () => {
+    const transformedModule = tsTransformModule(
```

Comment:
Do you think it makes sense to have one props like everywhere else? It will be easier if need to refactor it. Also pass customTransformers as is ;_)

Reply:
Sry I don't get what you mean :) I like the expressiveness of the tests instead of looping over configurations if you mean that. But it's correct that it's easier to change. What do you mean by pass customTransformers as is?

Reply:
```
tsTransformModule(props : ITsTransformModule) {}
```

like everywhere in the code, a function accepts only one argument. That's not about the test per se, just about the function itself ;-)

Reply:
Hmm, that changes quite alot. Why did you merge the initial PR in? ;)

Reply:
Then we'd basically change the interface, both calls, all tests  create the interface. Should we do that all in this PR?

Reply:
Ok, let's merge this one, and change after that ;-) I am ok with that. It's just better if we keep it consistent everywhere =)